Tests/ManualTests/ManualTestingApp/Singles/StatusBar.ux

```diff
@@ -0,0 +1,29 @@
+<Page Title="Status bar" ux:Class="StatusBar">
+	<InfoStack ux:Key="Info">
+		<p>This page is intended to test the status bar handling when a phone call is recieved with the app open.</p>
+	</InfoStack>
+	<StackPanel Margin="10">
```

Comment:
I think it'd be good to have something aligned to all the corners of the page. I guess the top frame already has the app header/buttons, but maybe put too circles in the bottom left/right corners to ensure they aren't getting clipped and/or distorted.